src/mame/drivers/spg2xx.cpp

```diff
@@ -1994,6 +2110,9 @@ CONS( 2009, senwfit,    0,        0, gssytts,   senwfit,   spg2xx_game_senwfit_s
 CONS( 2006, vtechtvssp, 0,        0, spg2xx,    spg2xx,    spg2xx_game_state,          empty_init,    "VTech",                                                  "TV Station (VTech, Spain)",                                             MACHINE_NOT_WORKING )
 CONS( 2006, vtechtvsgr, 0,        0, spg2xx,    spg2xx,    spg2xx_game_state,          empty_init,    "VTech",                                                  "TV Learning Station (VTech, Germany)",                                  MACHINE_NOT_WORKING )
 
+// "Boots's" is used on the title screen and in the manual, even if "Boots'" is usually used outside of this game.
+CONS( 200?, doraphon,   0,        0, doraphone, doraphone, spg2xx_game_doraphone_state,empty_init,    "VTech",                                                  "Dora the Explorer - Dora TV Explorer Phone / Boots's Special Day",      MACHINE_IMPERFECT_SOUND )
```

Comment:
Does it really have the error “Boots’s”?  The TV episode’s title is [Boots’ Special Day](https://dora.fandom.com/wiki/Boots%27_Special_Day) without the error.

Reply:
Yes, although I was unsure about this myself, and asked an English Language teacher from university, and they confirm that Boots's is correct in this context, and is what is displayed on the title screen (I thought it should be Boots' myself, but apparently that is incorrect)

Reply:
https://i.ibb.co/F3hLD3t/doraboots.png
see also the manual
https://www.manualslib.com/manual/411533/Vtech-Dora-S-Tv-Explorer-Phone.html?page=4#manual
"It's Boots's Special Day"

Reply:
It's Boots's here too on the DVD
https://www.youtube.com/watch?v=FkyWIqgFA80
I'm wondering if Boots's is actually the official representation, and the fan sites have tried to 'correct' it.